crates/lox-time/src/test_helpers.rs

```diff
@@ -1,11 +1,24 @@
+/*
+ * Copyright (c) 2024. Helge Eichhorn and the LOX contributors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#![cfg(test)]
```

Comment:
What's this?

Reply:
This marks the module as test-only. It serves the same purpose as

```rust
#[cfg(test)]
mod test_helpers
```

in the parent module, but communicates that fact to someone reading this file.